metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/DepublishRecordIdService.java

```diff
@@ -119,13 +120,15 @@ public DepublishRecordIdService(Authorizer authorizer, OrchestratorService orche
   }
 
   /**
-   * Adds a list of record ids to be depublished for the dataset.
+   * Deletes a list of record ids from the database. Only record ids that are in a {@link
+   * eu.europeana.metis.core.dataset.DepublishRecordId.DepublicationStatus#PENDING_DEPUBLICATION}
+   * state will be removed.
    *
    * @param metisUser The user performing this operation.
    * @param datasetId The ID of the dataset to which the depublished records belong.
    * @param recordIdsInSeparateLines The string containing the record IDs in separate lines.
-   * @return How many of the passed records were in fact added. This counter is not thread-safe: if
-   * multiple threads try to add the same records, their combined counters may overrepresent the
+   * @return How many of the passed records were in fact removed. This counter is not thread-safe:
```

Comment:
I think something went wrong with the documentation here. The method that this documentation is for is called `add...`

Reply:
Fixed